dot/network/service.go

```diff
@@ -334,19 +371,23 @@ func (s *Service) sentBlockIntervalTelemetry() {
 		if err != nil {
 			continue
 		}
+		bestHash := best.Hash()
+
 		finalized, err := s.blockState.GetFinalizedHeader(0, 0) //nolint
 		if err != nil {
 			continue
 		}
+		finalizedHash := finalized.Hash()
+
+		err = telemetry.GetInstance().SendMessage(telemetry.SystemIntervalTM{
+			BestHash:           &bestHash,
+			BestHeight:         best.Number,
+			FinalisedHash:      &finalizedHash,
+			FinalisedHeight:    finalized.Number,
+			TxCount:            big.NewInt(0), // todo (ed) determine where to get tx count
```

Comment:
is this regarding the tx pool? if so you can get the size via the `TransactionState`

Reply:
Ok, I've added interface to get Transactions Count.